cfme/common/__init__.py

```diff
@@ -355,26 +355,18 @@ def process_field(values):
             return map(SummaryValue, values)
 
 
-class Validatable(SummaryMixin):
-    """
-    Class which Middleware provider and other middleware pages must extend
-    to be able to validate properties values shown in summary page.
-    """
-
-    """
-    Tuples which first value is the provider class's attribute name,
-    the second value is provider's UI summary page field key.
-
-    Should have values in child classes.
+class Validatable(SummaryMixin, Taggable):
```

Comment:
Thanks for the observation, I haven't tested this, just put stuff together and did not realize this can happen too. I will remove this and rebase too.